components/common/logo/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+var curenetics_logo = require('./curenetics_logo_2.png');
```

Comment:
shouldn't have to require the logo, just do:

`<img src='./path-to-logo.png'>`